doc/md/fundamentals/4-control-flow/4-blocks.md

```diff
@@ -0,0 +1,65 @@
+---
+sidebar_position: 4
+---
+
+# Block expressions
+
+A block expression in Motoko is a sequence of statements enclosed in `{ ... }`. Blocks are used to group multiple operations, define local variables, and organize code for readability and logic.
+
+Block expressions are fundamental to function definitions, as they determine both the function’s structure and return value.
```

Comment:
```suggestion
Block expressions are fundamental to function definitions, as they determine both the function’s structure and return value.
```
```suggestion
Block expressions are typically used to define function and class bodies.
They can also be used as branches in conditional expressions and as the bodies of cases in switches and the arguments to `try-catch` expressions.

Since blocks  enclose declarations, the define new scopes for locally defined variables and types.
```